ts/screens/authentication/LandingScreen.tsx

```diff
@@ -208,6 +223,9 @@ class LandingScreen extends React.PureComponent<Props, State> {
         />
       );
     }
+    if (DeviceInfo.isTablet()) {
```

Comment:
Why the alert is shown inside the render?
Could it be called into `componentDidMount`?

Reply:
In the case `isRootedOrJailbroken` if called inside `componentDidMount` the alert is not displayed.